app/Http/Controllers/Backend/Social/TwitterController.php

```diff
@@ -92,16 +92,17 @@ public static function postStatus(Status $status): void {
             #dbl only works on Twitter.
             $socialText = $status->socialText;
             if ($status->user->always_dbl) {
-                $socialText .= " #dbl";
+                $socialText .= ' #dbl';
             }
-            $socialText .= ' ' . url("/status/{$status->id}");
-            $connection->post("statuses/update",
-                              [
-                                  "status" => $socialText,
-                                  'lat'    => $status->trainCheckin->Origin->latitude,
-                                  'lon'    => $status->trainCheckin->Origin->longitude
-                              ]
+            $socialText .= ' ' . url('/status/{$status->id}');
+            $response   = $connection->post('statuses/update',
+                                            [
+                                                'status' => $socialText,
+                                                'lat'    => $status->trainCheckin->Origin->latitude,
+                                                'lon'    => $status->trainCheckin->Origin->longitude
+                                            ]
             );
+            $status->update(['tweet_id' => $response->id]);
```

Comment:
I would expect this line under the 200 check (which should then `return` out). This way, we avoid an "unknown property" error if the http request turns out bad.